docs/ide.md

```diff
@@ -0,0 +1,29 @@
+# IDE Development
+
```

Comment:
This addition seems off-topic. I suggest to open a separated PR for IDE support.

Reply:
The PR is specifically to enable IDE development, so I documented how to use my changes. It could be a separate PR I suppose, but that seems a little too broken out for my usual tastes 🤔

Reply:
Sorry but the IDE part was not obvious from the title